pytorch_lightning/plugins/training_type/training_type_plugin.py

```diff
@@ -37,7 +37,7 @@
 TBroadcast = TypeVar("TBroadcast")
 
 
-class TrainingTypePlugin(ABC):
+class Strategy(ABC):
```

Comment:
the filename will be renamed later?

also, this isn't going to follow a deprecation process?

Reply:
This was still marked as experimental, not sure if we need one here

Reply:
Yes, (my) plan is to rename components first, then the files. 
Happy to hear suggestions how to "deprecate" things, but as Justus said it is inevitable that 1.6 will break people's custom plugins unfortunately, considering all the changes we already did. 

But we can deprecate things on an import level for example. People importing from `plugins.training_type` should get a warning that it was moved to `strategy.x`, maybe?

Reply:
if it's experimental then it's fine.
is it targeted to stabilize by v1.6?

Reply:
100%

Reply:
> Yes, our plan is to rename components first, then the files.

I would rather make this in single PR, as it trivial rename shall be fine

Reply:
Unfortunately this will result in 100+ files changed, nobody would review this. I would prefer to take it in smaller pieces.